### DISCUSSION TITLE
Official Chia Pool support

### DISCUSSION THREAD

**Discussion by guydavis** [guydavis/machinaris#153] [Announcements]
Hi all, we've been building/testing [official pool support in Machinaris](https://github.com/guydavis/machinaris/wiki/Pooling#official-chia-pools) for a week now.  To try it yourself, just pull the [test image](https://github.com/guydavis/machinaris/wiki/Releases#release-streams).

With the release of Chia 1.2.0 last night, we are just wrapping up testing.  Expect a final release of Machinaris v0.5.0 shortly. 

Please add your questions about official pools in this discussion thread.  Thanks in advance for your feedback!

**Comment by robertbigk** [comment]
OK can You tell me how download development version in unraid?

**Reply by guydavis** [reply]
In Unraid admin webui:

1. Click Docker tab
2. Click Machinaris application
3. Select Edit
4. On Machinaris edit screen, enter `ghcr.io/guydavis/machinaris:test`  for Repository line.
5. Click 'Done' button.

That will pull the test image and run it.  After final release **_in a few days_**, perform again but with `ghcr.io/guydavis/machinaris` (no test).

**Reply by robertbigk** [reply]
Wow thanks for some awesome and quick help

**Comment by robertbigk** [comment]
Ehh what I miss is not work?
![msedge_QNGbSTMxC2](https://user-images.githubusercontent.com/34956477/124944734-dc4a8500-e00d-11eb-8dad-175f598a3402.png)
![msedge_AYzlAOarSu](https://user-images.githubusercontent.com/34956477/124944806-eb313780-e00d-11eb-99ef-0d1561d97a1f.png)
![msedge_kMUj8g0zHK](https://user-images.githubusercontent.com/34956477/124944825-eec4be80-e00d-11eb-8eb7-9b08527b8e84.png)

I got menu test add repository test but chia wallet still 1.17

**Reply by guydavis** [reply]
Hi!  Looks like an old image pull on your system.  As you're on Unraid, be sure to 'Check for Updates' in the Docker tab and then apply them.

![image](https://user-images.githubusercontent.com/12936/124950511-b56d5e00-dfcf-11eb-99bd-72141420026e.png)
